src/helpers/performance-report-analyser.helper.js

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
+import chalk from 'chalk';
+
+class PerformanceReportAnalyser {
+  getReport(fileName) {
+    return JSON.parse(fs.readFileSync(`reports/performance/${fileName}`, 'utf8'));
+  }
+
+  parse(fileName) {
+    const reportFile = this.getReport(fileName);
+
+    return reportFile.log.entries.map(item => ({
+      ttfb: item.timings.wait,
+      url: item.request.url
+    }));
+  }
+
+  checkTiming(fileName, maxTiming) {
```

Comment:
I would change it to analyze, and rename a class to TimeToFirstByteAnalyzer - s or z (not sure what english we use, british or american)